src/ruby-rails-postgres/devcontainer-template.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "ruby-rails-postgres",
-    "version": "2.0.1",
+    "version": "3.0.0",
```

Comment:
I thought these changes would only bump minor version? Or are they really a breaking upgrade?
Just curious about the pattern

Reply:
These updates could be breaking eg. for Ruby, the default is now updated to a different one (which can be breaking, but versionings aren't currently used anywhere). Similarly, for EOLs, we stop providing some options which were present before.